argocd/assets/service_checks.json

```
@@ -58,5 +58,20 @@
         ],
         "name": "Application Controller cluster connection status",
         "description": "Returns `CRITICAL` if not connected to the cluster, otherwise returns `OK`."
+    },
+    {
+        "agent_version": "7.42.0",
```

nit: We can leave this for now. But ideally we should change it to try to match up for when it gets released with an agent.

```suggestion
        "agent_version": "7.42.0",
```

---

makes sense!